magichome/magichome.js

```diff
@@ -190,17 +168,16 @@ module.exports = function (RED) {
         }
 
         let internalState = new LampState();
-
         let node = this;
         let light = new MagicHomeControl(host);
 
         function setState(value) {
 
-            if (value === internalState.C_ON || value === true) {
+            if (value || value === internalState.C_ON) {
```

Comment:
truthy makes it prone to odd behaviour. Acutally even an msg.payload.power = "OFF" turns the light on, as value becomes truthy.